.azure-pipelines/docker.yml

```diff
@@ -0,0 +1,23 @@
+trigger:
+  branches:
+    include:
+      - 'task/add-acr-pipeline'
+  tags:
+    include:
+      - v*
+  paths:
+    include:
+      - src/**

```

Comment:
I think you should include additional paths that may affect the docker image like the `Dockerfile`

Reply:
I have removed this restriction to include all files.